llpc/context/llpcCompiler.cpp

```diff
@@ -2146,7 +2178,21 @@ void Compiler::LinkRelocatableShaderElf(
             }
         }
 
-        result = writer.LinkGraphicsRelocatableElf({&vsReader, &fsReader}, pContext);
+        if (!pShaderElfs[ShaderStageFetch].empty())
+        {
+            size_t codeSize = pShaderElfs[ShaderStageFetch].size_in_bytes();
+            result = fchReader.ReadFromBuffer(pShaderElfs[ShaderStageFetch].data(), &codeSize);
+            if (result != Result::Success)
+            {
+                return;
+            }
+            result = writer.LinkGraphicsRelocatableElf({&vsReader, &fsReader, &fchReader}, pContext);
+        }
+        else
+        {
+            result = writer.LinkGraphicsRelocatableElf({&vsReader, &fsReader}, pContext);
+        }
```

Comment:
Graphics pipelines are now always required to have the fetch shader stage, aren't they? So please convert the check into an assertion and remove the obsolete code paths.